contrib/docker/README.md

```diff
@@ -0,0 +1,91 @@
+# zcash Dockerfile
+
+This is an example Dockerfile to run the zcashd daemon in a containerized Debian base OS. The image pulls the apt package built by Electric Coin Company.
+
+The output of building this image should be accessible at https://hub.docker.com/repository/docker/electriccoinco/zcashd
```

Comment:
This url isn't accessible without logging into docker hub. Would a url like https://hub.docker.com/r/electriccoinco/zcashd be a good substitute here?

Reply:
Doh, updated!